uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/component/views/CheckoutView.kt

```diff
@@ -510,19 +498,35 @@ internal class CheckoutView @JvmOverloads constructor(context: Context,
         commentsListener?.invoke(commentBottomSheet)
     }
 
-    override fun bindTravelDetails(poiType: PoiType?){
+    override fun bindTravelDetails(poiType: PoiType?, travelDetails: String?){
         when(poiType){
             PoiType.AIRPORT -> {
                 bookingCheckoutTravelDetailsView.visibility = VISIBLE
                 bookingCheckoutTravelDetailsView.setActionIcon(R.drawable.kh_uisdk_ic_checkout_airport)
                 bookingCheckoutTravelDetailsView.setTitle(context.getString(R.string.kh_uisdk_checkout_airport_title))
                 bookingCheckoutTravelDetailsView.setSubtitle(context.getString(R.string.kh_uisdk_checkout_airport_subtitle))
+
+                travelDetails?.let {
+                    flightInfo = it
+                    if(it.isNotEmpty())
+                        bookingCheckoutTravelDetailsView.setSubtitle(it)
+                    else
+                        bookingCheckoutTravelDetailsView.setSubtitle(context.getString(R.string.kh_uisdk_checkout_airport_subtitle))
+                }
             }
             PoiType.TRAIN_STATION -> {
                 bookingCheckoutTravelDetailsView.visibility = VISIBLE
                 bookingCheckoutTravelDetailsView.setActionIcon(R.drawable.kh_uisdk_ic_checkout_train)
                 bookingCheckoutTravelDetailsView.setTitle(context.getString(R.string.kh_uisdk_checkout_train_title))
                 bookingCheckoutTravelDetailsView.setSubtitle(context.getString(R.string.kh_uisdk_checkout_train_subtitle))
```

Comment:
Lets refactor this whole bindTravelDetails to use a helper function since there's alot of duplicated code here